<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorrowBook</class>
 <widget class="QWidget" name="BorrowBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px</string>
   </property>
   <property name="text">
    <string>Mượn sách</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSearch">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSearch">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:12px</string>
   </property>
   <property name="text">
    <string>Người mượn</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxNguoiMuon">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:12px</string>
   </property>
   <property name="text">
    <string>Tên sách </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineBookName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:12px</string>
   </property>
   <property name="text">
    <string>Ngày mượn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBorrow">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrow</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateBorrow">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
